bot/src/keeper/setup.ts

```diff
@@ -0,0 +1,28 @@
+import getSigner, { createSigner, setSigner } from 'auctions-core/src/signer';
+import { KEEPER_MINIMUM_NET_PROFIT_DAI, KEEPER_WALLET_PRIVATE_KEY } from '../variables';
```

Comment:
Do we maybe want to add a separate variable for Surplus Auctions? That way the user can define minimums for each?

Currently the keeper does not run if `KEEPER_MINIMUM_NET_PROFIT_DAI` is not set. Maybe we want to introduce the following behavior:

`KEEPER_MINIMUM_NET_PROFIT_DAI` Is set = Keeper runs for collaterals. 

`KEEPER_MINIMUM_NET_PROFIT_DAI` is not set = Keeper does not run for collaterals.

And the same for the Surplus Auctions as well. This will allow the user to also determine which auctions the keeper should run on.

Reply:
Question: What would the new variable look like?